src/roi/Roi.ts

```diff
@@ -500,56 +517,90 @@ export class Roi {
+   * Computes the surface of the ROI, including the surface of the holes.
+   *
+   *
+   * @returns Surface including holes measured in pixels.
+   */
   get filledSurface() {
     return this.surface + this.holesInfo.surface;
   }
 
   /**
-   * Calculates solidity of the ROI
+   *The solidity describes the extent to which a shape is convex or concave.
+   * The solidity of a completely convex shape is 1, the farther the it deviates from 1, the greater the extent of concavity in the shape of the ROI.
+   *
+   * @returns Solidity value.
    */
   get solidity() {
     return this.surface / getConvexHull(this.getMask()).surface;
   }
-  //TODO Should be refactored to not need creating a new Mask
+  //TODO Should be refactored to not need creating a new Mask.
+
+  /**
+   *Computes convex hull. It is the smallest convex set that contains it.
+   *
+   * @see https://en.wikipedia.org/wiki/Convex_hull
+   * @returns Convex hull.
+   */
   get convexHull() {
     return this.#getComputed('convexHull', () => {
       return getConvexHull(this.getMask());
     });
   }
   /**
-   * Calculates minimum bounding rectangle
+   * Computes the minimum bounding rectangle.
+   * In digital image processing, the bounding box is merely the coordinates of the rectangular border that fully encloses a digital image when it is placed over a page, a canvas, a screen or other similar bidimensional background.
+   *
+   * @returns The minimum bounding rectangle.
    */
   get mbr() {
     return this.#getComputed('mbr', () => {
       return getMbr(this.getMask());
     });
   }
 
+  /*
+  * Computes roundness of ROI.
+  * Roundness is the measure of how closely the shape of an object approaches that of a mathematically perfect circle.
+ (See slide 24 https://static.horiba.com/fileadmin/Horiba/Products/Scientific/Particle_Characterization/Webinars/Slides/TE011.pdf) */
   get roundness() {
-    /*Slide 24 https://static.horiba.com/fileadmin/Horiba/Products/Scientific/Particle_Characterization/Webinars/Slides/TE011.pdf */
     return (4 * this.surface) / (Math.PI * this.feret.maxDiameter.length ** 2);
   }
+  /**
+   * This is not a diameter in its actual sense but the common basis of a group of diameters derived from the distance of two tangents to the contour of the particle in a well-defined orientation.
+   *  In simpler words, the method corresponds to the measurement by a slide gauge (slide gauge principle).
+   * In general it is defined as the distance between two parallel tangents of the particle at an arbitrary angle. The minimum Feret diameter is often used as the diameter equivalent to a sieve analysis.
+   *
+   * @returns The maximum and minimum Feret Diameters.
+   */
 
   get feret(): Feret {
     return this.#getComputed('feret', () => {
       return getFeret(this.getMask());
     });
   }
   /**
+   * A JSON object with all the data about ROI.
    *
-   * @returns calculated properties as one object
+   * @returns Calculated properties as one object.
```

Comment:
It is more than the calculated properties. I guess it's supposed to be "All ROI properties"